SU-SWS/stanford_basic
# D8CORE-1410: Back to site link

# READY FOR REVIEW

# Summary
- Placing a back to site button on the login page at /user

# Review By (Date)
when possible

# Urgency
Not urgent

# Steps to Test

1. Check out this branch
1. Navigate to /user/login
1. Back to site button should be on that page
1. Verify that link takes you to the homepage
1. Verify that it doesn't interfere with the Primary Tabs

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-1410
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)

- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
